d2networking/d2netpacket/packet_add_player.go

```diff
@@ -24,16 +25,28 @@ type AddPlayerPacket struct {
 // AddPlayerPacket with the data in given parameters.
 func CreateAddPlayerPacket(id, name string, x, y int, heroType d2enum.Hero,
```

Comment:
also minor, I think we are trying to standardize our factory function names as `NewXYZ` instead of `CreateXYZ`